docs/CategoryGuides/MiscCategories/AllMajorEndings.md

```diff
@@ -651,6 +651,6 @@ The Asgore fight and Flowey encounter are exactly the same as in the previous en
 
 > This is effectively identical to the post-Flowey part of the [True Pacifist](/CategoryGuides/MainCategories/TruePacifist "UTSR Guides - True Pacifist") speedrun.
 
-The ONLY difference between this and TPE as described in that link is that you start at Core End instead of Judgement hall. Other than that, everything is exactly the same.
+The ONLY difference between this and TPE as described by link is that you start at Core End instead of Judgement hall. Other than that, everything is exactly the same.
```

Comment:
"by that link" would be more correct. I would personally advise "by that route's guide" but either is acceptable.